cloudformation/eks/helm/values-mainnet.yaml

```
@@ -0,0 +1,210 @@
+global:
+  isPubnet: true
+  ephemeralDatabase: false
+  eventBroker:
+    type: "NONE"
```

```suggestion
    type: "SCHEDULER"
```
This was changed in 3.7.0. We deprecated `ENABLE_SCHEDULER: "true"` and introduced the type `SCHEDULER`.